Sources/SwiftProtobuf/Message+JSONAdditions.swift

```diff
@@ -93,3 +93,72 @@ public extension Message {
   }
 }
 
+
+public extension Array where Iterator.Element: Message {
```

Comment:
This PR looks good! But my main concern about this change is the extensions on `Array` itself. Even though they're constrained to `SwiftProtobuf.Message` here, what would happen if someone else extended `Array` in a non-mutually-exclusive way (such as without a generic constraint at all) to add something like `jsonString` or `init(jsonString:)`? My guess is that the most specifically constrained one would get called, but either way, something would be inaccessible.

Or maybe I'm just worried for nothing, because this is how extensions work in Swift today and there's no way to work around this (you can't selectively import extensions) as the language is designed today. The only other option I see is mangling the names (`protobufJSONString`), which also sucks because (1) we explicitly decided not to do that in the messages themselves so (2) it would make the names inconsistent.

Thoughts?